content/lua/concepts/strings/terms/len/len.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.len()' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Returns the number of characters of a string.' # Required; ideally under 150 characters and starts with a present-tense verb (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Code Foundations'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Numbers'
+  - 'Functions'
+  - 'Strings'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
```

Comment:
Modifications:

- Let's remove the comments from the metadata
- Let's change the description for uniqueness
- Let's add another entry to both `Subjects:` and `Tags:`

```suggestion
---
Title: '.len()'
Description: 'Returns the number of characters that exist in a specific string.'
Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Numbers'
  - 'Functions'
  - 'Strings'
  - 'Characters'
CatalogContent:
  - 'learn-lua'
  - 'paths/computer-science'
---
```